src/components/students-in-categories/StudentsInCategories.styles.ts

```diff
@@ -0,0 +1,126 @@
+import { TypographyVariantEnum } from '~/types'
+
+export const styles = {
+  cardContainer: {
+    width: '100%',
+    maxWidth: '460px',
+    minHeight: '400px',
+    borderRadius: '8px',
+    border: '1px',
+    borderColor: 'basic.lightGrey',
+    backgroundColor: 'basic.white',
+    p: { xs: '20px 41px 40px', sm: '20px 41px 68px' },
+    boxSizing: 'border-box',
+    mt: '75px'
+  },
+
```

Comment:
please remove the blank lines between the `styles` properties, as it is not the default style for our project